.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "app/src/main/cpp/vrb"]
 	path = app/src/main/cpp/vrb
 	url = https://github.com/igalia/vrb.git
+[submodule "app/src/main/cpp/tinygltf"]
+	path = app/src/main/cpp/tinygltf
+	url = https://github.com/Igalia/tinygltf.git
```

Comment:
Hmm why are we using a fork instead of pointing to the original repo?

Reply:
Because I had to add [a patch](https://github.com/Igalia/tinygltf/commit/a83211097dacb55742f68b965564170818cf5c6f) for one thing that was missing. Once it is upstream, we can consider switching to the original repo.